Packs/PaloAltoNetworks_SecurityAdvisories/Integrations/PaloAltoNetworksSecurityAdvisories/PaloAltoNetworksSecurityAdvisories.py

```diff
@@ -167,6 +167,57 @@ def get_advisories(client: Client, product: str, sort: str = "-date", severity:
 
 def advisory_to_indicator(advisory_dict: dict):
     """Convert the advisory dictionary into an indicator dictionary"""
+
+    fields = {}
+
+    if "references" in advisory_dict:
+        references: list = advisory_dict.get('references', {}).get('reference_data', {})
+        fields['publications'] = [
+            {
+                "link": x.get('url'),
+                "title": x.get('name'),
+                "source": x.get('refsource')
+            } for x in references]
+
+    fields['cvss'] = advisory_dict.get("impact", {}).get("cvss", {}).get("baseScore", "")
+    fields['cvssscore'] = advisory_dict.get("impact", {}).get("cvss", {}).get("baseScore", "")
+    fields['cvedescription'] = advisory_dict.get("description", {}).get("description_data", [])[0].get("value", "")
+    fields['cvssvector'] = advisory_dict.get("impact", {}).get("cvss", {}).get("vectorString", "")
+    fields['sourceoriginalseverity'] = advisory_dict.get("impact", {}).get("cvss", {}).get("baseSeverity", "")
+    fields['published'] = advisory_dict.get("CVE_data_meta", {}).get("DATE_PUBLIC", "")
+
+    if "impact" in advisory_dict and advisory_dict.get("impact", {}).get("cvss", {}).get("version", "") == '3.1':
+        fields['cvssversion'] = advisory_dict.get("impact", {}).get("cvss", {}).get("version", "")
+
+        all_cvss = []
+        base_metric = advisory_dict.get("impact", {})
+        # is this v3 cvss?
+        cvss_v3 = base_metric.get('cvss')
+        all_cvss.append({"cvssV3": cvss_v3})
+
+        # fills out the cvss_v3 table
+        cvss_v3_data = []
+        for k, v in cvss_v3.items():
+            cvss_v3_data.append(
+                {
+                    "metric": camel_case_to_underscore(k).replace("_", " ").title(),
+                    "values": v
+                }
+            )
+        fields['cvss3'] = cvss_v3_data
+
+        # fills out the cvsstable in default cve layout
+        cvss_data = []
+        for k, v in cvss_v3.items():
+            cvss_data.append(
+                {
+                    "metrics": camel_case_to_underscore(k).replace("_", " ").title(),
+                    "value": v
+                }
+            )
+        fields['cvsstable'] = cvss_data
+
+    """
```

Comment:
please remove the old `return` if it's not relevant anymore

Reply:
removed